alot/buffers.py

```diff
@@ -85,7 +85,7 @@ def rebuild(self):
             self.isinitialized = True
 
         lines = list()
-        displayedbuffers = filter(self.filtfun, self.ui.buffers)
```

Comment:
Ftr only:

> Comments from GvR:  filter and map should die and be subsumed into list comprehensions

https://www.python.org/dev/peps/pep-0279/

If the expressions are too complex to be concisely represented in a comprehension, using the `itertools` alternatives mentioned by @dcbaker instead of the built-ins seems to be recommended [citation needed].